### PR TITLE
feat: add endpoint to check if an user is member of a dao

### PR DESCRIPTION
**Pull request by WaDadidou**

### CODE REVIEW

**Review by Villaquiranm — APPROVED** [review]

File: api/dao/v1/dao.proto [hunk 1]
```diff
@@ -15,6 +16,15 @@ message DAOsResponse {
     repeated DAO daos = 1;
 }
 
+message IsUserDAOMemberRequest {
+    string network_id = 1;
+    string user_address = 2;
+    string dao_address = 3;
```
Comment (n0izn0iz) [inline api/dao/v1/dao.proto:22]:
```suggestion
    string user_id = 1;
    string dao_id = 2;
```

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

**Review by omarsy — APPROVED** [review]

File: api/dao/v1/dao.proto (hunk 1 above)
Comment (WaDadidou) [reply api/dao/v1/dao.proto:22]: Agree ! I was not sure

File: go/pkg/dao/dao_service.go [hunk 2]
```diff
@@ -74,3 +74,42 @@ func (s *DAOService) DAOs(ctx context.Context, req *daopb.DAOsRequest) (*daopb.D
 		Daos: pbdaos,
 	}, nil
 }
+
+func (s *DAOService) IsUserDAOMember(ctx context.Context, req *daopb.IsUserDAOMemberRequest) (*daopb.IsUserDAOMemberResponse, error) {
+	db := s.conf.IndexerDB
+	userId := req.GetUserId()
+	daoId := req.GetDaoId()
+
+	daoNetwork, daoAddress, parseDaoIdErr := s.conf.NetStore.ParseUserID(daoId)
```
Comment (n0izn0iz) [inline go/pkg/dao/dao_service.go:83]:
```suggestion
	daoNetwork, daoAddress, err := s.conf.NetStore.ParseUserID(daoId)
```

File: go/pkg/dao/dao_service.go [hunk 3]
```diff
@@ -74,3 +74,42 @@ func (s *DAOService) DAOs(ctx context.Context, req *daopb.DAOsRequest) (*daopb.D
...
+	if parseDaoIdErr != nil {
+		return nil,
+			errors.Wrap(parseDaoIdErr, "failed to parse dao id")
+	}
+
+	_, userAddress, parseUSerIdErr := s.conf.NetStore.ParseUserID(userId)
```
Comment (n0izn0iz) [inline go/pkg/dao/dao_service.go:89]:
```suggestion
	_, userAddress, err := s.conf.NetStore.ParseUserID(userId)
```

**Review by n0izn0iz — CHANGES_REQUESTED** [review]
unless you need to save an error for later use, naming all errors `err` is idiomatic go

**Review by n0izn0iz — APPROVED** [review]

**Review by omarsy — APPROVED** [review]

**Pull request merged**